ocsp/ocsp.go

```diff
@@ -101,8 +101,27 @@ func ReasonStringToCode(reason string) (reasonCode int, err error) {
 }
 
 // NewSignerFromFile reads the issuer cert, the responder cert and the responder key
-// from PEM files, and takes an interval in seconds
+// from PEM files, and takes an interval in seconds, and returns a Signer
 func NewSignerFromFile(issuerFile, responderFile, keyFile string, interval time.Duration) (Signer, error) {
+	log.Debug("Loading responder key: ", keyFile)
+	keyBytes, err := ioutil.ReadFile(keyFile)
+	if err != nil {
+		return nil, cferr.Wrap(cferr.CertificateError, cferr.ReadFailed, err)
+	}
+
+	key, err := helpers.ParsePrivateKeyPEM(keyBytes)
+	if err != nil {
+		log.Debug("Malformed private key %v", err)
+		return nil, err
+	}
+
+	return NewSignerFromFileAndKey(issuerFile, responderFile, key, interval)
+}
+
+// NewSignerFromFileAndKey reads the issuer cert and the responder cert
+// from PEM files, and takes a crypto signer an interval in seconds,
+// and returns a signer
+func NewSignerFromFileAndKey(issuerFile, responderFile string, key crypto.Signer, interval time.Duration) (Signer, error) {
```

Comment:
Was considering that name too, can make it so, not all that particular.. just.. it seemed too much like a "sea-shells-from-the-see" tongue twister "NewSigner..AndSigner"..